### DISCUSSION TITLE
Flux?

### DISCUSSION THREAD

**Discussion by Tenofas** [vladmandic/sdnext#3358] [Ideas]
Is there a chance SDNest will support also FLUX in the future?
Thanks.

**Comment by vladmandic** [comment]
Definitely.

**Reply by SAC020** [reply]
> Definitely.

Any tentative timeline, at least on the dev branch? Flux looks awesome and I can't wait to try it out in SDNext. 

Thanks

**Reply by vladmandic** [reply]
dev branch supports flux.
needs a bit of polishing to add auto-download and stuff like that, release likely in 10-15 days as I'm currently traveling.

**Reply by SAC020** [reply]
I understand Vlad is traveling, has anyone been able to load Flux - if yes, how?

I have downloaded flux1-dev.safetensors from huggingface, put it into /models/stable-diffusion, I am getting this error. I am obviously missing something but no idea what.

> 16:55:22-288491 INFO     Select: model="flux1-dev"
> 16:55:22-313787 DEBUG    Load model: existing=False target=C:\ai\automatic\models\Stable-diffusion\flux1-dev.safetensors info=None
> 16:55:23-286938 DEBUG    GC: utilization={'gpu': 58, 'ram': 1, 'threshold': 80} gc={'collected': 704, 'saved': 7.9} before={'gpu': 9.34, 'ram': 0.36} after={'gpu':
>                          1.44, 'ram': 0.71, 'retries': 0, 'oom': 0} device=cuda fn=unload_model_weights time=0.68
> 16:55:23-289915 DEBUG    Unload weights model: {'ram': {'used': 0.71, 'total': 63.92}, 'gpu': {'used': 1.44, 'total': 15.99}, 'retries': 0, 'oom': 0}
> 16:55:23-347946 DEBUG    Diffusers loading: path="C:\ai\automatic\models\Stable-diffusion\flux1-dev.safetensors"
> 16:55:23-349435 INFO     Autodetect: model="FLUX" class=FluxPipeline file="C:\ai\automatic\models\Stable-diffusion\flux1-dev.safetensors" size=22700MB
> 16:55:23-350924 ERROR    Diffusers model cannot load safetensor model: C:\ai\automatic\models\Stable-diffusion\flux1-dev.safetensors Autodetect
> 16:55:23-353403 DEBUG    Setting changed: sd_model_checkpoint=flux1-dev progress=True
> 16:55:23-357370 DEBUG    Save: file="config.json" json=32 bytes=1332 time=0.004
> 16:55:23-359850 DEBUG    Unused settings: ['cross_attention_options']

**Reply by nitinmukesh** [reply]
> I understand Vlad is traveling, has anyone been able to load Flux - if yes, how?
> 
> I have downloaded flux1-dev.safetensors from huggingface, put it into /models/stable-diffusion, I am getting this error. I am obviously missing something but no idea what.

AFAIK it does not go in s-d folder but unet folder, this is what I have seen in ComfyUI. Haven't tried as still waiting for official release.

**Reply by SAC020** [reply]
> > I understand Vlad is traveling, has anyone been able to load Flux - if yes, how?
> > I have downloaded flux1-dev.safetensors from huggingface, put it into /models/stable-diffusion, I am getting this error. I am obviously missing something but no idea what.
> 
> AFAIK it does not go in s-d folder but unet folder, this is what I have seen in ComfyUI. Haven't tried as still waiting for official release.

If I move the .safetensor into unet it's no longer available in the models drop-down list to load

**Reply by vladmandic** [reply]
it's coming later this week.
